integrationtest/batch-jsr352/src/test/java/org/hibernate/search/integrationtest/batch/jsr352/massindexing/MassIndexingJobWithMultiTenancyIT.java

```diff
@@ -11,42 +11,54 @@
 
 import java.io.IOException;
 import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import javax.batch.operations.JobOperator;
 import javax.batch.runtime.BatchRuntime;
 import javax.batch.runtime.BatchStatus;
 import javax.batch.runtime.JobExecution;
-import javax.persistence.EntityManagerFactory;
-import javax.persistence.Persistence;
 
 import org.hibernate.Session;
 import org.hibernate.SessionFactory;
 import org.hibernate.search.batch.jsr352.core.massindexing.MassIndexingJob;
 import org.hibernate.search.batch.jsr352.core.massindexing.test.util.JobTestUtil;
 import org.hibernate.search.integrationtest.batch.jsr352.massindexing.entity.Company;
 import org.hibernate.search.mapper.orm.Search;
+import org.hibernate.search.mapper.orm.automaticindexing.AutomaticIndexingStrategyName;
+import org.hibernate.search.mapper.orm.cfg.HibernateOrmMapperSettings;
+import org.hibernate.search.util.impl.integrationtest.common.rule.BackendConfiguration;
+import org.hibernate.search.util.impl.integrationtest.mapper.orm.OrmSetupHelper;
 
-import org.hibernate.testing.RequiresDialect;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
 
 /**
  * @author Mincong Huang
  */
-@RequiresDialect(
-		comment = "The connection provider for this test ignores configuration and requires H2",
-		strictMatching = true,
-		value = org.hibernate.dialect.H2Dialect.class
-)
 public class MassIndexingJobWithMultiTenancyIT {
 
+	@Rule
+	public OrmSetupHelper ormSetupHelper = OrmSetupHelper
+			.withSingleBackend( new BackendConfiguration() {
+				@Override
+				protected Map<String, Object> backendProperties() {
+					HashMap<String, Object> properties = new HashMap<>();
+					properties.put( "type", "lucene" );
+					properties.put( "type", "local-heap" );
+					properties.put( "multi_tenancy.strategy", "discriminator" );
+					return properties;
+				}
+			} );
```

Comment:
Just do this, so that indexes are stored on disk, but in the right directory:

```suggestion
			.withSingleBackend( new LuceneBackendConfiguration() );
```

(you may need to add a dependency to `hibernate-search-util-internal-integrationtest-backend-lucene`)


Then add the `multi_tenancy.strategy` option below, in the `setUp` method:

```
.withBackendProperty( "multi_tenancy.strategy", "discriminator" )
```